.github/workflows/e2e_libvirt.yaml

```diff
@@ -79,6 +82,7 @@ jobs:
       - name: Config Libvirt
         run: |
           ./libvirt/config_libvirt.sh
+          echo "CAA_IMAGE=\"${{ inputs.caa_image }}\"" >> libvirt.properties
```

Comment:
Hi @stevenhorsman, maybe a dump question, but as I'm learning the project, I'm finding hard to understand why `libvirt.properties`?

Will this work only for libvirt? Should we use something like `$TEST_PROVISION_FILE` for a more agnostic approach?

Reply:
So this is specifically the libvirt e2e tests, so `libvirt.properties` is the file that config_libvirt.sh sets some info too and I'm just adding to it: https://github.com/confidential-containers/cloud-api-adaptor/blob/0db87d03d6887884d2c27a0b86dccc1a51bfa8f7/src/cloud-api-adaptor/libvirt/config_libvirt.sh#L115-L117
We then set `export TEST_PROVISION_FILE="$PWD/libvirt.properties"` later in this workflow.

Reply:
Got it. thanks!